k8s/eks/yaml/drop-weave-cm.yml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
```

Comment:
Combination of [this](https://github.com/testground/testground/issues/1455) and [this](https://github.com/testground/testground/issues/1456) issue.
Note that the issue described in 1455 also applies to the VPC CNI / weave combination.

Reply:
YAML supports comments so why don't we drop that knowledge inline? On another note though, how do these issues relate to the api version? I'm a bit confused.

Reply:
Added a comment in the yaml.
It does not have anything to do with the apiversion, I just left a comment on the 1st line in the file.